File: TodayFitComplete_Backend/src/main/java/com/SJY/TodayFitComplete_Backend/dto/request/member/MemberLoginDto.java
package com.SJY.TodayFitComplete_Backend.dto.request.member;

import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * 로그인 요청 DTO.
 */
@Getter
@Setter
@NoArgsConstructor
public class MemberLoginDto {
    private String email;
    private String password;
}

File: TodayFitComplete_Frontend/app/src/main/java/kr/ac/kumoh/ce/s20190633/todayfitcomplete_frontend/Dto/Member/MemberLoginDto.kt
package kr.ac.kumoh.ce.s20190633.todayfitcomplete_frontend.Dto.Member

// MemberLoginDto 클래스: 사용자 로그인에 필요한 데이터를 나타내는 데이터 전송 객체 (DTO)
data class MemberLoginDto(
    val email: String,     // 사용자 이메일 주소
    val password: String   // 사용자 비밀번호
)